dexcom/device.go

```diff
@@ -19,10 +19,14 @@ const (
 	DeviceDisplayDeviceShareReceiver       = "shareReceiver"
 	DeviceDisplayDeviceTouchscreenReceiver = "touchscreenReceiver"
 
-	DeviceTransmitterGenerationG4    = "g4"
-	DeviceTransmitterGenerationG5    = "g5"
-	DeviceTransmitterGenerationG6    = "g6"
-	DeviceTransmitterGenerationG6Pro = "g6 pro"
```

Comment:
no listed `g6 pro` as per dexcom spec ... but might have to keep an eye out

Reply:
Yup, we found that empirically - it wasn't in the v2 spec, either.

Reply:
interestingly `"g4" "g5" "g6"` are the only documented ones for the v2 spec

Reply:
FYI `g6 pro` listed with a note that it isn't listed in the spec